datajunction-server/alembic/env.py

```diff
@@ -85,13 +85,14 @@ def run_migrations_online():
     and associate a connection with the context.
 
     """
-    connectable = create_engine(settings.index)
+    connectable = create_engine(settings.index,connect_args={"options": "-csearch_path=ex2"})
```

Comment:
Are the options passed in here something that you can make configurable through `.env` instead? It's also not clear why you would want to hardcode this to ex2.